pingcap/tidb-operator
# charts: persist checkpoint for TiDB-Lightning

### What problem does this PR solve?

close #3620

### What is changed and how does it work?

set `checkpoint.dsn` under the source data directory.

### Code changes


### Tests


  - import with `keep-after-success = true`, and verify that a checkpoint file is generated and kept in the source data directory
  - import with `keep-after-success = false`, and verify that no checkpoint file kept after success

### Side effects


### Related changes


### Release Notes

Please refer to [Release Notes Language Style Guide](https://github.com/pingcap/tidb-operator/blob/master/docs/release-note-guide.md) before writing the release note.

```release-note
Supports to persist checkpoint for TiDB-Lightning helm chart
```
